M2/Macaulay2/m2/CMakeLists.txt

```diff
@@ -29,4 +30,8 @@ foreach(filename IN LISTS M2LIST ITEMS
   file(COPY ${filename} DESTINATION ${CORE_DIR})
 endforeach()
 
+## Copy Core tests from Macaulay2/tests/normal to share/doc/Macaulay2/Core/tests
+file(COPY ${CMAKE_SOURCE_DIR}/Macaulay2/tests/normal/ DESTINATION ${CORE_TESTS}
```

Comment:
I'd tried to do this in https://github.com/Macaulay2/M2/commit/7a55f01d5c7cc3c89612b168a4ff82e652aa03ad, but I guess it wasn't working properly?  We probably should revert that commit, too.